base/tools/src/main/java/com/netscape/cmstools/kra/KRAKeyFindCLI.java

```diff
@@ -78,6 +78,10 @@ public void createOptions() {
         option.setArgName("realm");
         options.addOption(option);
 
+        option = new Option(null, "ownerName", true, "ownerName: <tokenID>:<userID>");
+        option.setArgName("tokenID:userID");
```

Comment:
This should be changed to `owner` as well.

Reply:
fixed.